errors/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "errors"
+version = "0.0.1"
+authors = ["C4DT", "RustCrypto Developers"]
+edition = "2018"
+license = "Apache-2.0 OR MIT"
+homepage = "https://github.com/RustCrypto/nacl-compat"
+repository = "https://github.com/RustCrypto/nacl-compat/tree/master/errors"
```

Comment:
I'm not sure having a crate for common error types is a good idea, and regardless the name `errors` is already taken (and not exactly descriptive).

I'd suggest having an errors module per-crate, even if that leads to some duplication.